examples/99_docs/run_hyperparameters_docs.py

```diff
@@ -244,21 +244,121 @@
 # cross-validation.
 #
 # By default, ``julearn`` uses a :class:`~sklearn.model_selection.GridSearchCV`.
-# This searcher is very simple. First, it construct the "grid" of
+# This searcher, specified as "" is very simple. First, it construct the "grid" of
 # hyperparameters to try. As we see above, we have 3 hyperparameters to tune.
 # So it constructs a 3-dimentional grid with all the possible combinations of
 # the hyperparameters values. The second step is to perform cross-validation
 # on each of the possible combinations of hyperparameters values.
 #
-# Another searcher that ``julearn`` provides is the
-# :class:`~sklearn.model_selection.RandomizedSearchCV`. This searcher is
-# similar to the :class:`~sklearn.model_selection.GridSearchCV`, but instead
+# Another searchers that ``julearn`` provides are the
+# :class:`~sklearn.model_selection.RandomizedSearchCV` and 
+# :class:`~skopt.BayesSearchCV`. 
+#
+# The randomized searcher is similar to the
+# :class:`~sklearn.model_selection.GridSearchCV`, but instead
 # of trying all the possible combinations of hyperparameters values, it tries
 # a random subset of them. This is useful when we have a lot of hyperparameters
 # to tune, since it can be very time consuming to try all the possible, as well
 # as continuous parameters that can be sampled out of a distribution. For
 # more information, see the
 # :class:`~sklearn.model_selection.RandomizedSearchCV` documentation.
+#
+# The Bayesian searcher is a bit more complex. It uses a Bayesian optimization
```

Comment:
- Maybe also link the class for "Bayesian searcher" here?
- ... uses a Bayesian ... -> ... uses Bayesian ...